content/applications/inventory_and_mrp/inventory/routes/strategies/putaway.rst

```diff
@@ -1,54 +1,242 @@
+
+To create a :guilabel:`Request for Quotation` (RFQ), go to the :guilabel:`Purchase` app, and click
+:guilabel:`Create`. Then, add products to the :abbr:`RFQ (Request for Quotation)` by clicking
+:guilabel:`Add a line`, and selecting the desired product(s) from the drop-down menu in the
+:guilabel:`Product` column.
 
-Now, if I purchase apples and carrots to my supplier, they will be grouped in the same receipt but
-redirected to the right location automatically, thanks to putaway rules. This information is
-available from *Inventory Report*, under the reporting menu.
+Once ready, click :guilabel:`Confirm`. Doing so turns the :abbr:`RFQ (Request for Quotation)` into a
+:abbr:`PO (Purchase Order)`. Next, click :guilabel:`Send by Email` to send the :abbr:`PO (Purchase
+Order)` to the vendor if needed.
 
-.. image:: putaway/putaw3.png
+.. image:: putaway/purchase-apples-and-carrots.png
    :align: center
+   :alt: Purchase order for apples and carrots
 
-.. image:: putaway/putaw4.png
+Begin receiving the incoming shipment by selecting the :guilabel:`Receipt` smart button on the
+:abbr:`PO (Purchase Order)`. Click :menuselection:`Receive products --> Validate` to move products
+into the warehouse. Since the putaway rule is set to trigger when products arrive in stock, internal
+transfers are automatically created when products arrive in `WH/Stock`. This ensures products
+entering the warehouse are stored in specific locations suitable for their storage needs.
+
+.. image:: putaway/receive-incoming-stock.png
    :align: center
+   :alt: Receive package of apples and carrots
+
+Verify stock moves
+==================
+
+Confirm whether the putaway rule worked as intended and that the products have been moved to the
+correct location using the :guilabel:`Inventory Report`. View all products in stock in
+:menuselection:`Inventory app --> Reporting --> Inventory Report`. By default, the products are
+grouped by location.
 
-.. image:: putaway/putaw5.png
+Select a product, and the drop-down menu displays the locations where the products currently are.
+Select the location drop-down menu to view a list of all stock movements in and out of the location.
+Each of these lines are records of a stock move, called a *stock move line*.
+
+.. image:: putaway/destination-locations.png
    :align: center
+   :alt: Verify stock moves triggered by putaway rule
+
+.. _inventory/putaway/wrong-location:
+
+Trace products in wrong locations
+---------------------------------
+
+To trace products and associated documents for warehouse operations, use the :guilabel:`Product
+Moves` report in :menuselection:`Inventory app --> Reporting --> Product Moves`. View a product's
+warehouse operation history by selecting it from the product drop-down. Each stock move line
+contains details about any movement of stock, such as manufacturing orders, inventory adjustments,
+or internal transfers. Click on a stock move line to view all the information in a comprehensive
+report.
```

Comment:
I went through this, and since product is a list item, and guilabel:`Product` is a column, I don't think guilabels on the product would be accurate to use to refer to the list o: What do you think?